pylint.cfg

```diff
@@ -1,2 +1,2 @@
 [MESSAGES CONTROL]
-disable=C0103,E1101,I1101,R0801
+disable=C0103,E1101,I1101,R0801,E0606,W0719
```

Comment:
Also please remove `E0606`. I have read why you added it but unfortunately don't find the variable in question as the line numbers you give don't seem to match (probably outdated).

In any case, there is almost _always_ a better way to ensure that `E0606` is not raised. And even if not, I still find it better to disable it per line or per block, not globally. In the config we only want to put rules that we generally disagree with, in (almost) all cases.

Reply:
I put it globally because it was affecting other scripts and as that particular script will be modified in another PR I though of taking this shortcut to avoid blocking everything, but I can fix it here directly as the original PR was to fix the static code analysis.

Reply:
While `E0606` can give false positives, using variables before assignment is always a critical error - so if you disable this check globally, it becomes much easier to introduce such bugs accidentally. Quite a price to pay - especially since usually it is very easy to avoid the warning. If you tell me the code in question, I can give you advice on how to avoid it.